src/Features/Blockcore.Features.RPC/RPCClient.cs

```diff
@@ -1403,11 +1403,11 @@ public async Task<decimal> EstimatePriorityAsync(int nblock)
         /// <param name="commentTx">A locally-stored (not broadcast) comment assigned to this transaction. Default is no comment</param>
         /// <param name="commentDest">A locally-stored (not broadcast) comment assigned to this transaction. Meant to be used for describing who the payment was sent to. Default is no comment</param>
         /// <returns>The TXID of the sent transaction</returns>
-        public uint256 SendToAddress(BitcoinAddress address, Money amount, string commentTx = null, string commentDest = null)
+        public uint256 SendToAddress(BitcoinAddress address, Money amount, string commentTx = null, string commentDest = null, decimal fee = 0, bool isSegwit = false)
```

Comment:
Could we do nullable instead of 0? Not sure though.

Reply:
Ah yeah man, that makes sense.

Reply:
> Could we do nullable instead of 0? Not sure though.

So we can't default a decimal to null, but we could do -1? or just keep it at 0

Reply:
this is a nullable decimal:
`decimal? yourvar = null`

Reply:
> this is a nullable decimal:
> `decimal? yourvar = null`

Done.